static/js/typeahead_helper.js

```diff
@@ -103,7 +103,64 @@ exports.sorter = function (query, objs, get_item) {
 exports.compare_by_pms = function (user_a, user_b) {
     var count_a = people.get_recipient_count(user_a);
     var count_b = people.get_recipient_count(user_b);
-
+    var current_subject = $('#subject.recipient_box').val();
+    var current_subject_personsv = {};
+    var current_stream = $('#stream.recipient_box').val();
+    function pSubject(cMessage){
```

Comment:
I think function name should something clearer as well, probably, `filterSubject`.